src/seedu/addressbook/data/AddressBook.java

```diff
@@ -22,13 +24,19 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
+    private final ArrayList<Tagging> allTaggings;
 
     /**
      * Creates an empty address book.
      */
     public AddressBook() {
         allPersons = new UniquePersonList();
         allTags = new UniqueTagList();
+        allTaggings = new ArrayList<Tagging>();
+    }
+    
+    public ArrayList<Tagging> getAllTaggings() {
```

Comment:
Where is the header comment for this method. Please have it as a practice to add comments.